client/src/App.tsx

```diff
@@ -1,11 +1,34 @@
-import { BlockListContainer, ExtrinsicListContainer } from './components';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+// blockList
+import BlockListContainer from "BlockList";
+// extrinsicList
+import ExtrinsicListContainer from "ExtrinsicList";
+// layout
+import {
+  MainLayout,
+  MainLayoutContainer,
+  MainLayoutFooter,
+  MainLayoutHeader,
+} from "layout";
+// home
+import Home from "Home";
 
 function App() {
   return (
-    <div>
-      <BlockListContainer />
-      <ExtrinsicListContainer />
-    </div>
+    <BrowserRouter>
+      <MainLayout>
+        <MainLayoutHeader />
+        <MainLayoutContainer>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="block" element={<BlockListContainer />} />
```

Comment:
I would suggest storing paths in constants and reference values instead of having these as strings because it's easy to make typos

Reply:
similar to `EXTERNAL_ROUTES` we already have